infra/chromadb_compaction/lambdas/enhanced_compaction_handler.py

```
@@ -0,0 +1,1679 @@
+        except Exception as e:  # pylint: disable=broad-exception-caught
+            logger.error("Error parsing SQS message", error=str(e))
+            
+            if OBSERVABILITY_AVAILABLE:
+                metrics.count("CompactionMessageParseError", 1, {
+                    "error_type": type(e).__name__
+                })
+            continue
+
+    # Process stream messages if any
+    if stream_messages:
+        result = process_stream_messages(stream_messages)
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)